library/ord-tree.lisp

```diff
@@ -407,7 +419,9 @@ The result tree may be in an intermediate state with a double-black node."
       ((Branch clr left pivot right)
        (let (Tuple new-right removed-max) = (remove-max right))
        (Tuple (bubble-double-black clr left pivot new-right)
-              removed-max))))
+              removed-max))
+      (_
```

Comment:
Same as previous two.

Reply:
This one is also missing `Empty`

Reply:
This function currently is not exported and is only called by `remove-leaving-double-black` on non-empty trees, so it could have `(_ (error "Attempt to `remove-max` from an empty tree"))` like previous, but we might want to export this operation, in which case it should return an `Optional (Tuple (Tree :elt) :elt)`, return `None` in the `(Empty)` case, and error in the `(DoubleBlackEmpty)` case. But that change seems outside the scope of this PR, so I suggest the catch-all error.